lib/optimizely.rb

```diff
@@ -523,6 +524,47 @@ def close
       @event_processor.stop! if @event_processor.respond_to?(:stop!)
     end
 
+    def get_optimizely_config # rubocop:disable Naming/AccessorMethodName
+      # Get OptimizelyConfig object containing experiments and features data
+      # Returns Object
+      #
+      # OptimizelyConfig Object Schema
+      # {
+      #   'experimentsMap' => {
+      #     '111111' => {
```

Comment:
The examples are misleading as "id"-based map. Change them all to "key"-based map.

Reply:
Great catch. Fixed!. I copied them over from javascript-sdk. will fix it there too.